style.css

```diff
@@ -621,3 +642,529 @@
   width: 35.73%;
   margin-left: 32.135%;
 }
+
+@media only screen and (min-width: 768px) {
+  * {
```

Comment:
- [ ] In the process of making the app responsive (@ exactly 768px) as suggested earlier, I observed some things that came up and we need to adjust:

1.  There is no space between the `nav` items.
2.  Multi-Post Stories text and button are overlapped.
3.  The Project cards text are overlapped with other content.
4.  The About Me background-image is overlapped with the text.

Kindly implement all these to give your app a great User Interface. 👍